qa/L0_simple_lib/test.sh

```diff
@@ -85,37 +85,13 @@ for SIMPLE_CLIENT in simple ; do
         set -e
     done
 
-    # custom model needs to be obtained elsewhere
-    rm -rf $MODELSDIR
+    # Use savedmodel for addsub ensemble
     mkdir -p $MODELSDIR/simple/1
-    cp -r ../custom_models/custom_float32_float32_float32/1/* $MODELSDIR/simple/1/.
-    cp ../custom_models/custom_float32_float32_float32/config.pbtxt $MODELSDIR/simple/.
+    cp -r $DATADIR/savedmodel_float32_float32_float32/1/* $MODELSDIR/simple/1/.
+    cp $DATADIR/savedmodel_float32_float32_float32/config.pbtxt $MODELSDIR/simple/.
     (cd $MODELSDIR/simple && \
             sed -i "s/^name:.*/name: \"simple\"/" config.pbtxt && \
             sed -i "s/label_filename:.*//" config.pbtxt)
-    full=custom_float32_float32_float32
```

Comment:
Why are these removed?

Reply:
The custom_float32_float32_float32 is an addsub model that used the legacy addsub.so. Since we do not have a replacement for it yet we have to remove it. As we discussed we do not want to add a new addsub backend right?